starfish/image/_constants.py

```diff
@@ -0,0 +1,8 @@
+class Keys(object):
+    X_COORDS = 'x'
+    Y_COORDS = 'y'
+    Z_COORDS = 'z'
+
+    HYB = 'hyb'
+    CH = 'ch'
+    ZLAYER = 'zlayer'
```

Comment:
rename to Z

Reply:
Actually, we could do:

```
class CoordinateKey(object):
    X = 'x'
    Y = 'y'
    Z = 'z'

class IndexKey(object):
    HYB = 'hyb'
    CH = 'ch'
    Z = 'z'
```

it would disambiguate between the two.

Reply:
Would that be ok with you @dganguli ?

Reply:
love it. i would drop the Key though, e.g, CoordinateKey -> Coordinates and IndexKey -> Indices or Axes. Key is somehow the wrong word for both dimensions/indices/axes and coordinates

Reply:
it is primarily used as a key to a dict, but not always, i guess.  ok, i'll make the change.